src/dom_entity_names/events/event_names.rs

```diff
@@ -72,7 +72,9 @@ make_events! {
     DeviceOrientation => "deviceorientation",
     OrientationChange => "orientationchange",
     SmartcardInsert => "smartcard-insert",
-    SmartCardRemove => "smartcard-remove"
+    SmartCardRemove => "smartcard-remove",
+
+    GoPointerCapture => "gotpointercapture",
```

Comment:
GoPointer.. vs GotPointer.. (typo?)

Reply:
yes :P, I' ll correct it

Reply:
I was very tired yesterday I guess :P , I put as resolved :)